docs/essentials/collections/hashtableandDictionaryCollectionTypes.md

```diff
@@ -1 +1,32 @@
+# Hashtable and Dictionary Collection Types
+
+The [System.Collections.Hashtable](http://dotnet.github.io/api/System.Collections.Hashtable.html) class, and the [System.Collections.Generic.Dictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.Dictionary%602.html) and [System.Collections.Concurrent.ConcurrentDictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.ConcurrentDictionary%602.html) generic classes implement the [System.Collections.IDictionary](http://dotnet.github.io/api/System.Collections.IDictionary.html) interface. The **Dictionary&lt;TKey, TValue&gt;** generic class also implements the [IDictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.IDictionary%602.html) generic interface. Therefore, each element in these collections is a key-and-value pair.
```

Comment:
@cartermp In MSDN we typically use bold to denote language keywords or framework members if we aren't linking. The code style provided by the backticks is typically used for inline code.

We should be consistent across Core docs - let's discuss at Friday's meeting. I think we can merge as-is and fix later if necessary.